archive/settings.py

```diff
@@ -234,6 +239,18 @@
 PROCESSED_EXCHANGE_ENABLED = ast.literal_eval(os.getenv('PROCESSED_EXCHANGE_ENABLED', 'True'))
 PROCESSED_EXCHANGE_NAME = os.getenv('PROCESSED_EXCHANGE_NAME', 'archived_fits')
 
+# Additional Customization
+ZIP_DOWNLOAD_FILENAME_BASE = os.getenv('ZIP_DOWNLOAD_FILENAME_BASE', 'ocs_archive_data')
+ZIP_DOWNLOAD_MAX_UNCOMPRESSED_FILES = os.getenv('ZIP_DOWNLOAD_MAX_UNCOMPRESSED_FILES', 10)
```

Comment:
Should this be cast as an integer too? I believe environment variables are strings by default.

Reply:
you're not gonna let me download 1.5 files?